sample-apps/react-native/dogfood/src/components/ActiveCall.tsx

```diff
@@ -1,36 +1,41 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 import {
-  CallContentProps,
   CallControls,
   CallControlsType,
   CallingState,
-  ParticipantsInfoBadge,
   CallContent,
   useCall,
   useIncallManager,
   theme,
+  CallTopView,
 } from '@stream-io/video-react-native-sdk';
-import { ActivityIndicator, StyleSheet, View } from 'react-native';
+import { ActivityIndicator, StyleSheet } from 'react-native';
 import { appTheme } from '../theme';
 import {
   SafeAreaView,
   useSafeAreaInsets,
 } from 'react-native-safe-area-context';
 import { ActiveCallNotification } from './ActiveCallNotification';
-import { ParticipantsLayoutSwitchButton } from './ParticipantsLayoutButton';
+import { ParticipantsInfoList } from './ParticipantsInfoList';
 
-type ActiveCallProps = CallControlsType;
-
-type Layout = CallContentProps['mode'];
+type ActiveCallProps = CallControlsType & {
+  onBackPressed?: () => void;
+};
 
 export const ActiveCall = ({
   chatButton,
   hangUpCallButton,
+  onBackPressed,
 }: ActiveCallProps) => {
   const call = useCall();
   const activeCallRef = useRef(call);
   activeCallRef.current = call;
-  const [selectedLayout, setSelectedLayout] = useState<Layout>('grid');
+  const [isCallParticipantsVisible, setIsCallParticipantsVisible] =
+    useState<boolean>(false);
+
+  const onOpenCallParticipantsInfo = useCallback(() => {
+    setIsCallParticipantsVisible(true);
+  }, [setIsCallParticipantsVisible]);
```

Comment:
can we remove the `useCallback` here, it is unnecessary

Reply:
Done ✅